### DISCUSSION TITLE
Scheduled Maintenance 🛠️

### DISCUSSION THREAD

**Discussion by mikhail-tokarev** [codemagic-ci-cd/codemagic-docs#1466] [Announcements]
On **March 13, 2023** we will perform a firewall upgrade to increase network throughput for all macOS machines. The maintenance window is between **13:00 - 14:00 UTC** and the expected downtime is approximately 10-15 minutes.

**Comment by 0ttik** [comment]
Maintenance window is over, but the builds are still not starting and support doesn't reply.

**Comment by thilojaeggi** [comment]
Yeah still down. It's 14:23 in UTC.

**Reply by joaovsa** [reply]
Still down 😢

**Comment by rafaelhaguiar** [comment]
Still not working, queued.

**Comment by Piyushhhhh** [comment]
Builds are still queued

**Comment by helinanever** [comment]
Update: The firewall has been upgraded but there are issues with accessing the build machines which causes longer build queues. We are working with our data center engineers to resolve this as soon as possible. Thank you for your patience!

**Reply by helinanever** [reply]
RESOLVED: The connection issues were caused by a configuration problem with the new firewall. This has been fixed now. However, it may take some time for all queued builds to start.

**Reply by rafaelhaguiar** [reply]
For me still not working, i tried run again, but stay in queue

**Comment by mrruby** [comment]
Working now for me

**Comment by mcoquet** [comment]
Builds are failing for me. 403 when pulling source code from AWS code commit

**Reply by mikhail-tokarev** [reply]
Hi, please start a new thread since this error most probably related to something else
